internal/runners/activate/activate.go

```diff
@@ -149,7 +149,8 @@ func (r *Activate) run(params *ActivateParams) error {
 		if params.Branch != proj.BranchName() {
 			return locale.NewInputError(
 				"err_conflicting_branch_while_checkedout",
-				"Cannot activate branch [NOTICE]{{.V0}}[/RESET]; Branch [NOTICE]{{.V1}}[/RESET] is already checked out.",
+				"Cannot activate branch [NOTICE]{{.V0}}[/RESET]. Branch [NOTICE]{{.V1}}[/RESET] is already checked out.\n"+
+					"Please use [ACTIONABLE]state branch switch <name>[/RESET] followed by [ACTIONABLE]state activate[/RESET] instead.",
```

Comment:
The comment makes sense but if it's going to be multiple lines or just longer than others we should move it to the localization file.